app/views/publish/providers/schools/delete.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= render PageTitle.new(title: "support.providers.schools.delete") %>
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(publish_provider_recruitment_cycle_school_path(@provider.provider_code, @site.recruitment_cycle.year, @site.id)) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-l"><%= @site.location_name %></span>
+    <% if Course.includes(:sites).where(sites: { id: @site.id}).count.zero? %>
```

Comment:
Two things:

- Indentation looks a little off with this erb 
- This should probably go into a helper/or somewhere under a friendly method name rather than leaking model db methods into the view

Reply:
Done